integration-tests/src/test/java/tachyon/worker/BlockServiceHandlerIntegrationTest.java

```diff
@@ -0,0 +1,285 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.worker;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+
+import org.apache.thrift.TException;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import tachyon.Constants;
+import tachyon.TachyonURI;
+import tachyon.TestUtils;
+import tachyon.client.OutStream;
+import tachyon.client.TachyonFS;
+import tachyon.client.TachyonFSTestUtils;
+import tachyon.client.WriteType;
+import tachyon.conf.TachyonConf;
+import tachyon.master.LocalTachyonCluster;
+import tachyon.master.MasterInfo;
+import tachyon.thrift.ClientFileInfo;
+import tachyon.thrift.FileDoesNotExistException;
+import tachyon.thrift.InvalidPathException;
+import tachyon.thrift.OutOfSpaceException;
+import tachyon.underfs.UnderFileSystem;
+import tachyon.util.CommonUtils;
+import tachyon.worker.block.BlockServiceHandler;
+import tachyon.worker.block.io.BlockWriter;
+
+/**
+ * Integration tests for tachyon.BlockServiceHandler
+ */
+public class BlockServiceHandlerIntegrationTest {
+  private static final long WORKER_CAPACITY_BYTES = 10000;
+  private static final long USER_ID = 1L;
+  private static final int USER_QUOTA_UNIT_BYTES = 100;
+
+  private LocalTachyonCluster mLocalTachyonCluster = null;
+  private MasterInfo mMasterInfo = null;
+  private BlockServiceHandler mWorkerServiceHandler = null;
+  private TachyonFS mTfs = null;
+  private TachyonConf mMasterTachyonConf;
+  private TachyonConf mWorkerTachyonConf;
+
+  @After
+  public final void after() throws Exception {
```

Comment:
It seems to me this method should be `afterClass` and the same for `beforeClass`